buildozer/targets/android.py

```diff
@@ -47,26 +47,28 @@
 # does.
 DEFAULT_SDK_TAG = '4333796'
 
+DEFAULT_ARCH = 'armeabi-v7a'
```

Comment:
Actually at some point maybe the default should be `arm64-v8a` since the play store requires at least that one when uploading APK

Reply:
I agree, but chose not to change the default in this particular PR.